app/helpers/pieces_helper.rb

```diff
@@ -1,2 +1,18 @@
 module PiecesHelper
+  def render_piece_show(x, y)
+    piece = @game.pieces.find_by(position_x: x, position_y: y)
+
+    if piece == nil
+      simple_form_for(@current_piece, url: piece_path(@current_piece)) do |f|
+        f.input :position_x, as: :hidden, input_html: {value: x} 
+        f.input :position_y, as: :hidden, input_html: {value: y}
+        f.submit 'Update'
+      end
```

Comment:
Getting snagged somewhere in the above simple_form. appears I'm not passing through the positions? threw in a pry below and can see `piece` has all of the info of the selected piece i.e. `@current_piece`